### PR TITLE
add application version manager service wrapper

### PR DESCRIPTION
**Pull request by HookedBehemoth**

### CODE REVIEW

File: nx/include/switch/services/avm.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * @author Behemoth
+ * @copyright libnx Authors
+ */
+#pragma once
+
+#include "../types.h"
+#include "../sf/service.h"
+
+typedef struct {
+    u64 app_id;
```
Comment (yellows8) [inline nx/include/switch/services/avm.h:13]: `application_id`, same below

File: nx/source/services/avm.c [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    );
+}
+
+Result avmGetLaunchRequiredVersion(u64 application_id, u32 *version) {
+    return serviceDispatchInOut(&g_AvmSrv, 200, application_id, *version);
+}
+
+static Result _avmPushVersion(u32 cmd_id, u64 application_id, u32 version) {
+    const struct {
+        u64 application_id;
```
Comment (yellows8) [inline nx/source/services/avm.c:63]: Is this param order really correct?

File: nx/source/services/avm.c [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+void _avmCleanup(void) {
+    serviceClose(&g_AvmSrv);
+}
+
+Service *avmGetServiceSession(void) {
+    return &g_AvmSrv;
+}
+
+static Result _avmGetVersion(u32 cmd_id, u64 id_1, u64 id_2, u32 *version) {
```
Comment (yellows8) [inline nx/source/services/avm.c:30]: cmd_id param should be last, same below.

File: nx/source/services/avm.c (hunk 2 above)
Comment (HookedBehemoth) [reply nx/source/services/avm.c:63]: whoops... happened while merging commands ^^

**Review by yellows8 — APPROVED** [review]

**Review by fincs — APPROVED** [review]

**Pull request merged**